test/steer/histogram.xml

```diff
@@ -1,14 +1,21 @@
 <?xml version="1.0" enconding="us-ascii"?>
 
-<marlin xmlns:xsi="https://www.w3.org/2001/XMLShema-instance" xsi:noNamespaceSchemaLocation="htts://ilcsoft.desy.de/marlin/marlin.xsd">
+<marlin>
   <execute>
     <processor name="MyTestHistogram" />
   </execute>
 
   <global>
-    <parameter name="Verbosity" options="DEBUG0-4,MESSAGE0-4,WARING0-4,ERROR0-4,SILENT"> MESSAGE3 DEBUG </parameter>
+    <parameter name="Verbosity" options="DEBUG0-4,MESSAGE0-4,WARING0-4,ERROR0-4,SILENT"> DEBUG5 </parameter>
+    <parameter name="Concurrency">3</parameter>
   </global>
 
+  <bookstore>
+    <parameter name="OutputFile">histTest.root</parameter>
+    <parameter name="DefaultMemoryLayout">Copy</parameter>
+    <parameter name="StoreByDefault">YES</parameter>
```

Comment:
Use a boolean here, `true` or `false`. The default would be if you don't specify anything

Reply:
Flag will be removed.